src/components/shared/Description.tsx

```diff
@@ -1,13 +1,47 @@
 import { MarkdownViewer } from '@stoplight/markdown-viewer';
-import { Box } from '@stoplight/mosaic';
+import { Box, Link, Text } from '@stoplight/mosaic';
 import * as React from 'react';
 
-export const Description: React.FunctionComponent<{ value: string }> = ({ value }) => (
-  <Box
-    as={MarkdownViewer}
-    markdown={value}
-    style={{
-      fontSize: 12,
-    }}
-  />
-);
+export const Description: React.FunctionComponent<{ value: string }> = ({ value }) => {
+  const [showAll, setShowAll] = React.useState(false);
+
+  const paragraphs = value.split('\n\n');
+
+  if (paragraphs.length <= 1 || showAll) {
+    return (
+      <Box
+        as={MarkdownViewer}
+        markdown={value}
+        style={{
+          fontSize: 12,
+        }}
+      />
+    );
+  }
+
+  const firstParagraph = paragraphs[0];
+
+  return (
+    <Box
+      as={MarkdownViewer}
+      markdown={firstParagraph}
+      parseOptions={{
+        components: {
+          p: (props: any) => {
+            return (
+              <Box as="p">
+                <Text mr={1}>{props.children}</Text>
+                <Link cursor="pointer" onClick={() => setShowAll(true)}>
```

Comment:
isn't this a button?

Reply:
I could add `role="button"`, but link works just fine as a clickable and tabbable thing 🤷‍♂️. I don't feel strongly about it, but might be annoying to render a button here that doesn't have button styling.

Reply:
If we want to be more correct semantically this would be a button, but I get a different styling would be applied, so meh.